src/main/scala/uk/gov/nationalarchives/tdr/api/utils/TreeNodesUtils.scala

```diff
@@ -1,32 +1,32 @@
 package uk.gov.nationalarchives.tdr.api.utils
 
-import java.io.{File => JIOFile}
-import java.util.UUID
-import uk.gov.nationalarchives.tdr.api.model.file.NodeType.{fileTypeIdentifier, folderTypeIdentifier}
+import uk.gov.nationalarchives.tdr.api.model.file.NodeType.directoryTypeIdentifier
 import uk.gov.nationalarchives.tdr.api.service.UUIDSource
 import uk.gov.nationalarchives.tdr.api.utils.TreeNodesUtils.TreeNode
 
+import java.io.{File => JIOFile}
+import java.util.UUID
 import scala.annotation.tailrec
 
 class TreeNodesUtils(uuidSource: UUIDSource) {
-  def generateNodes(filePaths: Set[String]): Map[String, TreeNode] = {
-    @tailrec
-    def innerFunction(originalPath: String, fileType: String, nodes: Map[String, TreeNode]): Map[String, TreeNode] = {
-      val jioFile = new JIOFile(originalPath)
-      val parentPath = Option(jioFile.getParent)
-      val name = jioFile.getName
-      val treeNode = TreeNode(uuidSource.uuid, name, parentPath, fileType)
-      val nextMap = nodes + (originalPath -> treeNode)
-      if (parentPath.isEmpty) {
-        nextMap
-      } else {
-        innerFunction(parentPath.get, folderTypeIdentifier, nextMap)
-      }
+  @tailrec
+  private def innerFunction(originalPath: String, fileType: String, nodes: Map[String, TreeNode]): Map[String, TreeNode] = {
```

Comment:
Maybe change the `fileType` parameter to something else? Not sure what though?

Reply:
I've called it `typeIdentifier` to match what the calling method calls it.